TrackingTools/GsfTracking/python/GsfElectronTracking_cff.py

```diff
@@ -10,3 +10,11 @@
 from TrackingTools.GsfTracking.CkfElectronCandidateMaker_cff import *
 from TrackingTools.GsfTracking.GsfElectronGsfFit_cff import *
 electronGsfTracking = cms.Sequence(electronSeeds*electronCkfTrackCandidates*electronGsfTracks)
+
+from SimTracker.TrackAssociation.trackTimeValueMapProducer_cfi import trackTimeValueMapProducer
+gsfTrackTimeValueMapProducer = trackTimeValueMapProducer.clone(trackSrc = cms.InputTag('electronGsfTracks'))
+
+electronGsfTrackingWithTiming = cms.Sequence(electronGsfTracking.copy()*gsfTrackTimeValueMapProducer)
```

Comment:
please define with leading underscore (``` _electronGsfTrackingWithTiming ``` ) so that this doesn't propagate to other eras and the reference sequence name renames the same